doc/source/docker/run-quickstart-examples-docker-compose.rst

```diff
@@ -0,0 +1,175 @@
+Run Flower Quickstart Examples with Docker Compose
+==================================================
+
+Flower provides a set of `quickstart examples <https://github.com/adap/flower/tree/main/examples>`_
+to help you get started with the framework.
+
+These examples are designed to demonstrate the capabilities of Flower and can be run
+using Docker Compose.
+
+.. important::
+
+   Some quickstart examples may have limitations or requirements that prevent them from running
+   on every environment. For more information, please see `Limitation`_.
+
+Prerequisites
+-------------
+
+Before you start, make sure that:
+
+- The ``flwr`` CLI is :doc:`installed <../how-to-install-flower>` locally.
+- The Docker daemon is running.
+- Docker Compose is `installed <https://docs.docker.com/compose/install/>`_.
+- Two open terminal windows.
+
+**Why Two Terminals?**
+
+It is recommended to use two terminals to run the quickstart examples. This allows you to keep the
+``docker compose`` commands in one terminal and the ``flwr`` commands in the other, avoiding the
+need to switch between directories.
+
+Step 1: Start the Docker Services (Terminal 1)
+----------------------------------------------
+
+#. Clone the Flower Repository:
+
+   .. code-block:: bash
+
+      $ git clone --depth=1 https://github.com/adap/flower.git
+
+
+#. Change the directory to the location of the Docker Compose file:
+
+   .. code-block:: bash
+
+      $ cd flower/src/docker/complete/
+
+#. Set the ``PROJECT_DIR`` environment variable to the path of the quickstart example you want to
+   run. The path should be relative to the location of the Docker Compose files:
+
+   .. code-block:: bash
+
+      $ export PROJECT_DIR=../../../examples/quickstart-sklearn-tabular
+
+#. Build and start the services using the following command:
+
+   .. code-block:: bash
+
+      $ docker compose -f compose.yml up --build -d
+
+#. Follow the logs of the SuperExec service:
+
+   .. code-block:: bash
+
+      $ docker compose logs superexec -f
+
+Step 2: Run the Quickstart Example (Terminal 2)
+-----------------------------------------------
+
+#. Change the directory to the quickstart example:
+
+   .. code-block:: bash
+
+      $ cd flower/examples/quickstart-sklearn-tabular
+
+#. Append the following lines to the end of the ``pyproject.toml`` file and save it:
+
+   .. code-block:: toml
+      :caption: pyproject.toml
+
+      [tool.flwr.federations.docker-compose]
```

Comment:
We could add this to the `pyproject.toml` file to remove this step. wdyt?

Reply:
I think leaving it for people to do could be a simple but useful learning experience for future projects?

Reply:
I think it is ok to do it for a single example but i personally got a little bit annoyed when i tried different examples.